src/components/followedAccounts/accountTitleInput.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import Input from '../toolbox/inputs/input';
+import styles from './followedAccounts.css';
+
+const AccountTitleInput = ({
+  t, title, onChange, className, disabled, hideLabel,
+}) => {
+  const validateInput = value => (value.length > 20 ? t('Title too long') : undefined);
+
+  return <Input
+            label={hideLabel ? null : t('Title (optional)')}
+            className={`${styles.titleInput} ${className} account-title `}
+            error={title.error}
+            value={title.value}
+            autoFocus={true}
```

Comment:
If autoFocus is always true you can put just `autofocus` instead of `autoFocus={true}`